src/components/BuildConfigDetailPage/BuilConfigRevisionPage.tsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react';
+
+import { useParamsRequired } from 'hooks/useParamsRequired';
+import { useServiceContainer } from 'hooks/useServiceContainer';
+
+import { BuildConfigDetail } from 'components/BuildConfigDetailPage/BuildConfigDetail';
+
+import * as buildConfigApi from 'services/buildConfigApi';
+
+export const BuildConfigRevisionPage = () => {
+  const { buildConfigId, revisionId } = useParamsRequired();
+
+  const serviceContainerBuildConfigRevision = useServiceContainer(buildConfigApi.getRevision);
+  const serviceContainerRevisionRunner = serviceContainerBuildConfigRevision.run;
```

Comment:
[optional] I would rather use full version `serviceContainerBuildConfigRevisionRunner`, it's useful when we are doing searching and expecting `serviceContainerBuildConfigRevision` part is included, you are using the full version also for example here: BuildConfigRevisionPages.tsx

Reply:
true, that was intended by me